packages/api/vitest.config.mts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    sequence: {
+      hooks: 'list',
+    },
+    setupFiles: [
+      './vitest.setup.mts'
+    ],
+    logHeapUsage: true,
```

Comment:
Love this option:

<img width="974" alt="image" src="https://github.com/redwoodjs/redwood/assets/32992335/da5463ce-d5e4-4788-afc8-ff676ea1cfc0">